docs/reference.md

```diff
@@ -570,10 +570,11 @@ Currently, the following footprint types are supported:
 ## Phew, that's it.
 
 *Theoretically*, you should know everything to start making your own dream keyboard.
-*Realistically*, tho, this might have been a bit dense, to say the least.
+*Realistically* though, this might have been a bit dense, to say the least.  If you aren't confused enough and want to look at an example that leverages the full power of Ergogen, you can find [the Absoluem project here](https://github.com/mrzealot/absolem/) and its [configuration file here.](https://github.com/mrzealot/absolem/blob/master/absolem.yaml) 
```

Comment:
I'd switch the new sentence (If you aren't...) with the one below (But hey...), plus there's a small typo in Absolem. Looks good otherwise!

Reply:
I think the latest change does what you're asking for.  I also tweaked the language to be less self-deprecating since ideally once a few changes have been implemented it should give users a good idea on how to use the project.  Feel free to let me know if you'd like it to be moved back to the previous verbiage.